frontend/src/hooks/useCreateBoard.tsx

```diff
@@ -97,6 +97,10 @@ const useCreateBoard = (team: Team, stakeHolders: string[]) => {
 	);
 
 	const canAdd = useMemo(() => {
+		if (dividedBoardsCount === teamMembers.length) {
+			return false;
+		}
+
 		if (dividedBoardsCount === Math.floor(teamMembers.length / 2)) {
 			return false;
 		}
```

Comment:
Why not make the two conditions in one if, something like:
```suggestion
		if ((dividedBoardsCount === teamMembers.length) || (dividedBoardsCount === Math.floor(teamMembers.length / 2))) {
			return false;
		}
```